tutorials/cloud_access/euclid-cloud-access.md

```diff
@@ -277,10 +277,10 @@ spec_association_tbl
 If you picked a target other than what this notebook uses, it's possible that there is no spectrum associated for your target's object ID. In that case, `spec_association_tbl` will contain 0 rows.
 ```
 
-In above table, we can see that the `uri` column gives us location of spectra file on IBE. We can map it to S3 bucket key to retrieve spectra file from the cloud. This is a very big FITS spectra file with multiple extensions where each extension contains spectrum of one object. The `hdu` column gives us the extension number for our object. So let's extract both of these.
+In above table, we can see that the `path` column gives us a url that can be used to call the SpectrumDM service to get the spectrum of our object. We can map it to an S3 bucket key to retrieve a spectra file from the cloud. This is a very big FITS spectra file with multiple extensions where each extension contains spectrum of one object. The `hdu` column gives us the extension number for our object. So let's extract both of these.
```

Comment:
> a url that can be used to call the SpectrumDM service 

Is SpectrumDM a service though? My understanding is that it's the data model and that the service will take a fits file url and turn it into a votable that follows the SpectrumDM standard -- nitpicking, but may be worth being clear and explicit in the explanation